lib/api.js

```diff
@@ -1,6 +1,6 @@
 module.exports = adminsApi
 
-var clone = require('lodash.clone')
+var clone = require('lodash')
```

Comment:
Yeah the case is not covered by tests, I’ll make an issue to bring this module to 100% test coverage :)